sdk/storage/azure-storage-blob/azure/storage/blob/_serialize.py

```diff
@@ -51,7 +51,8 @@
     '2021-02-12',
     '2021-04-10',
     '2021-06-08',
-    '2021-08-06'
+    '2021-08-06',
+    '2021-12-02'
```

Comment:
Slightly concerned with this change + the fact that I had to use preprod for these tests to work. I'm assuming the pipeline is not going to be very happy?

Reply:
Yep, the Live test pipeline will not be happy against this PR, but ci should be fine due to recordings. That's fine for the private preview and the reason this can't be merged to `main`